docs/restful-interface.rst

```diff
@@ -1,499 +1,400 @@
+          },
+          "literature": {
+              "PUB00020128": {
+                  "PMID": 15284453,
+                  "ISBN": null,
+                  "volume": "305",
+                  "issue": "5689",
+                  "year": 2004,
+                  "title": "Crystal structure of Argonaute and its implications for RISC slicer activity.",
+                  "URL": null,
+                  "raw_pages": "1434-7",
+                  "medline_journal": "Science",
+                  "ISO_journal": "Science",
+                  "authors": [
+                      "Song JJ",
+                      "Smith SK",
+                      "Hannon GJ",
+                      "Joshua-Tor L."
+                  ],
+                  "DOI_URL": "http://dx.doi.org/10.1126/science.1102514"
+              },
+              "PUB00018283": {
+                  "PMID": 11050429,
+                  "ISBN": null,
+                  "volume": "25",
+                  "issue": "10",
+                  "year": 2000,
+                  "title": "Domains in gene silencing and cell differentiation proteins: the novel PAZ domain and redefinition of the Piwi domain.",
+                  "URL": null,
+                  "raw_pages": "481-2",
+                  "medline_journal": "Trends Biochem Sci",
+                  "ISO_journal": "Trends Biochem. Sci.",
+                  "authors": [
+                      "Cerutti L",
+                      "Mian N",
+                      "Bateman A."
+                  ],
+                  "DOI_URL": "http://dx.doi.org/10.1016/S0968-0004(00)01641-8"
+              }
+          },
+          "set_info": {
+              "accession": "CL0219",
+              "name": "RNase_H"
+          },
+          "overlaps_with": null,
+          "counters": {
+              "subfamilies": 0,
+              "domain_architectures": 602,
+              "interactions": 0,
+              "matches": 29456,
+              "pathways": 0,
+              "proteins": 28420,
+              "proteomes": 2266,
+              "sets": 1,
+              "structural_models": {
+                  "alphafold": 21504,
+                  "rosettafold": 0
+              },
+              "structures": 112,
+              "taxa": 9708
+          },
+          "entry_annotations": {
+              "hmm": 0,
+              "logo": 0,
+              "alignment:uniprot": 22300,
+              "alignment:full": 14038,
+              "alignment:seed": 15
+          },
+          "cross_references": {}
+      }
+  }
 
-Using curl
-^^^^^^^^^^
+Some Pfam families are removed or merged into others, in which case they become "dead" families. 
+If you try to retrieve annotation information about a dead family, you'll get a simple JSON document that only tells you that there 
+aren't any content associated to this entry.
```

Comment:
typo

Reply:
what's the typo?

Reply:
that's not a typo, in markdown you can go to the next line, to avoid having the lines too long.